src/commands/gist.ts

```diff
@@ -402,6 +402,43 @@ export async function registerGistCommands(context: ExtensionContext) {
     )
   );
 
+  context.subscriptions.push(
+    commands.registerCommand(
+      `${EXTENSION_NAME}.viewForkedGist`,
+      async (node?: StarredGistNode | FollowedUserGistNode) => {
+        await ensureAuthenticated();
+
+        if (node) {
+          const forks = await getForks(node.gist.id);
+
+          const items = sortGists(forks).map((g) => ({
+            label: getGistLabel(g),
```

Comment:
The `owner.login` property of the gist would include the owner's username, so maybe we should display this as `${g.owner.login} / ${getGistLabel(g)`?

Reply:
Good catch! I'll use this instead.